styles/pr-commit-view.less

```diff
@@ -42,12 +41,12 @@
     padding: 0;
     background-color: transparent;
     border: none;
+    cursor: pointer;
+    color: @text-color-highlight;
```

Comment:
@annthurium I changed this again. 😇 Using `@text-color-subtle` made it feel like the commits are disabled. And also harder to read because of low contrast. I assume your change was to make the hover state stronger. So now I changed it to the opposite -> default: strong, hover: subtle

![hover](https://user-images.githubusercontent.com/378023/50065883-22475f80-01fc-11e9-8b31-87fe82ceb6a7.gif)

Is that ok? I was also considering using an underline, but that we mostly use for "links" that open a browser.